bionic/utils/urls.py

```diff
@@ -36,7 +36,7 @@ def path_from_url(url):
 
 
 def url_from_path(path):
-    return Path(path).as_uri()
+    return unquote(Path(path).as_uri())
```

Comment:
If we unquote it here, then it's no longer a valid URL, right? Wouldn't it make more sense to unquote it in `path_from_url`?

Reply:
I think unquoted urls are technically valid urls too. But it makes sense to unquote it from `path_from_url` instead!

Reply:
I was going based on [this post](https://stackoverflow.com/questions/497908/is-a-url-allowed-to-contain-a-space), which states that spaces must be encoded in URLs.

BTW, I just remembered we also need a release note.

Reply:
TIL!